_episodes/11-joins.md

```diff
@@ -29,6 +29,10 @@ In this episode we will consider different scenarios and show we might join the
 
 ### Scenario 1 - Two data sets containing the same columns but different rows of data
 
+First, let's download the datafiles. You can download the ![pandas_join_types](../fig/pandas_join_types.png)
+![github repo](https://github.com/datacarpentry/python-socialsci/archive/gh-pages.zip); the data files are in the *data* directory. You'll need to place these files in the same directory you launch Python or Jupyter from.
```

Comment:
Downloading zip of whole lesson might be confusing and may add to the cognitive load.
Would it be possible to perhaps refer to the files from the setup sections?